sw.js

```diff
@@ -1,103 +1,47 @@
 importScripts(
-  'js/vendor/sw-cacheable-response.min.js',
-  'js/vendor/sw-routing.min.js',
-  'js/vendor/sw-runtime-caching.min.js',
-  'js/vendor/sw-precaching.min.js'
+  'https://storage.googleapis.com/workbox-cdn/releases/3.3.1/workbox-sw.js'
 );
 
-// Make this always match package.json version
-const version = '0.1.3';
-const cacheName = `defaultCache_${version}`;
-
-const { cacheableResponse, precaching, routing, runtimeCaching } = goog;
-
-const router = new routing.Router();
-const localhost = registration.scope;
-const staticCache = new precaching.UnrevisionedCacheManager({ cacheName });
-const requestWrapper = new runtimeCaching.RequestWrapper({ cacheName });
-const cdnRequestWrapper = new runtimeCaching.RequestWrapper({
-  plugins: [
-    new cacheableResponse.CacheableResponsePlugin({ statuses: [0, 200] })
-  ],
-  cacheName
-});
-
-/**
- * Route for local CSS and JS assets
- *
- * Strategy:
- * https://developers.google.com/web/fundamentals/instant-and-offline/
- * offline-cookbook/#stale-while-revalidate
- */
-const assetRoute = new routing.RegExpRoute({
-  regExp: new RegExp(`^${localhost}.*\\.(css|js|png|svg)$`),
-  handler: new runtimeCaching.StaleWhileRevalidate({ requestWrapper })
-});
+// Cache fallback for HTML files
+workbox.routing.registerRoute(
+  /(?:^[^.]+|\.html)$/,
+  workbox.strategies.networkFirst()
+);
 
-/**
- * Route for assets on cloudfront.net CDN
- *
- * Strategy:
- * https://developers.google.com/web/fundamentals/instant-and-offline/
- * offline-cookbook/#stale-while-revalidate
- */
-const cdnAssetRoute = new routing.RegExpRoute({
-  regExp: new RegExp('^https://.*\\.cloudfront\\.net/.*\\.(png|svg)$'),
-  handler: new runtimeCaching.StaleWhileRevalidate({
-    requestWrapper: cdnRequestWrapper
+// Cache CSS and JS files
+workbox.routing.registerRoute(
+  /\.(?:js|css)$/,
+  workbox.strategies.staleWhileRevalidate({
+    cacheName: 'static-resources'
   })
-});
-
-/**
- * Route for pages
- *
- * Strategy:
- * https://developers.google.com/web/fundamentals/instant-and-offline/
- * offline-cookbook/#network-falling-back-to-cache
- */
-const navRoute = new routing.NavigationRoute({
-  whitelist: [/./],
-  handler: new runtimeCaching.NetworkFirst({ requestWrapper })
-});
-
-/**
- * Precache resources
- */
-staticCache.addToCacheList({
-  unrevisionedFiles: ['/', '/404/', '/error/']
```

Comment:
It looks like the service worker was precaching resources before. Let's make sure the new service worker does the same. :)

Here are some docs about precaching files: https://developers.google.com/web/tools/workbox/guides/precache-files/

Reply:
You can see the difference in action by going offline via the DevTools and trying to hit the 404 page:
<img width="1479" alt="screen shot 2018-07-12 at 4 10 09 pm" src="https://user-images.githubusercontent.com/459757/42664183-3e18f9f6-85ee-11e8-8ca1-86d33649979e.png">